sycl/test-e2e/bindless_images/vulkan_interop/unsampled_images.cpp

```diff
@@ -433,12 +705,12 @@ int main() {
     return EXIT_FAILURE;
   }
 
-  bool validated = run_test();
+  auto exit_value = run_all();
 
   if (vkutil::cleanup() != VK_SUCCESS) {
     std::cerr << "Cleanup failed!\n";
     return EXIT_FAILURE;
   }
 
-  return validated ? EXIT_SUCCESS : EXIT_FAILURE;
+  return !exit_value;
```

Comment:
This is very strange... Can we rename the variable?

Reply:
changed to `run_ok`.